platform/linux-generic/include/odp_packet_io_internal.h

```diff
@@ -231,6 +235,18 @@ static inline void pktio_cls_enabled_set(pktio_entry_t *entry, int ena)
 	entry->s.enabled.cls = !!ena;
 }
 
+static inline int _odp_pktio_tx_ts_enabled(pktio_entry_t *entry)
+{
+	return entry->s.enabled.tx_ts;
+}
+
+static inline void _odp_pktio_tx_ts_set(pktio_entry_t *entry)
```

Comment:
This might be more useful in the future if it allowed the devide code to specify the time, e.g. by getting time from odp_time_global() only when time is not provided by the caller.

Reply:
I'd prefer adding the extra parameter if/when it's actually needed. This shouldn't be a complex change if the function's external functionality stays the same.